packages/document-service-api/test/developer/developer.test.js

```diff
@@ -1,15 +1,74 @@
+const config = require('../../src/configuration/config');
 const { documentTypes } = require('@pins/common'); // TODO: remove this when the document types from @pins/common are brought into this API.
 const supertest = require('supertest');
 const app = require('../../src/app');
 const api = supertest(app);
 const path = require('path');
 const each = require('jest-each').default;
 const { isBlobInStorage } = require('./testcontainer-helpers/azurite-container-helper');
+const { createPrismaClient } = require('../../src/db/db-client');
+const { seedStaticData } = require('@pins/database/src/seed/data-static');
+const { seedDev } = require('@pins/database/src/seed/data-dev');
+const { appealDocuments } = require('@pins/database/src/seed/appeal-documents-dev');
+const blobClient = require('#lib/back-office-storage-client');
+const fs = require('fs');
+
+jest.setTimeout(20000);
 
 jest.mock('../../src/configuration/featureFlag', () => ({
 	isFeatureActive: () => true
 }));
 
+jest.mock('express-oauth2-jwt-bearer', () => {
+	return {
+		auth: jest.fn(() => {
+			return (req, res, next) => {
+				req.auth = {
+					payload: {
+						sub: '29670d0f-c4b4-4047-8ee0-d62b93e91a11'
```

Comment:
couldn't figure out how to vary this by test

Reply:
I think the options are to either remock the entire function or you could try setting the auth header on the supertest request, but that might not work

Reply:
Oh or we could have a common func got getting the sub from a request then we mock that function, like
```
exports.getSub = (req) => req.auth.payload.sub
```
```
jest.mock('../getSub', () => ({
  getSub: () => <some guid>
}))
```

then you can just mock the return value

```
getSub.mockReturnValueOnce(<some other guid>)
```